<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmallScrewdriver</class>
 <widget class="QWidget" name="SmallScrewdriver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Упаковщик текстур</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/main/screwdriver.png</normaloff>:/main/screwdriver.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Кнопачки</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="go">
          <property name="text">
           <string>Поехали!</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/main/screwdriver.png</normaloff>:/main/screwdriver.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>709</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="methodGroupBox">
       <property name="title">
        <string>Метод</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="methodComboBox">
          <item>
           <property name="text">
            <string>Shelf Next Fit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Shelf First Fit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Guillotine</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Max Rects</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="binSizeGroupBox">
       <property name="title">
        <string>Размер атласов</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="binSizeComboBox">
          <item>
           <property name="text">
            <string>256 x 256</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>512 x 512</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1024 x 1024</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2048 x 2048</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4096 x 4096</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8192 x 8192</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Масштаб</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDoubleSpinBox" name="scaleSpinBox">
          <property name="minimum">
           <double>0.050000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.020000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
